dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/FileUploadTask.kt

```diff
@@ -151,7 +151,10 @@ abstract class FileUploadTask @Inject constructor(
         }
 
         val mappingFiles = getFilesList()
-        if (mappingFiles.isEmpty()) return
+        if (mappingFiles.isEmpty()) {
+            LOGGER.info("No mapping files to upload.")
```

Comment:
> [!NOTE]
> Should it be a warning instead of info ? By default, `info` level logs are not visible when running gradle. My guess is that running this task, mapping files are expected to exist, which could be a good enough reason to make the log a warning.

Reply:
Yes, fair point. Updated.